src/types.ts

```diff
@@ -203,3 +203,11 @@ export interface Organisation {
 }
 
 export interface UserOrg extends User, Organisation {}
+
+export interface PermissionDetails {
+  id: number
+  name: string
+  description: string
```

Comment:
Shouldn't `displayName: string` also be a property on here?

Reply:
I'm not actually sure how you got away with not having this and not getting a typescript error when you added it to the object 😄